### DISCUSSION TITLE
Local supabase-nextjs-user-management with Supabase CLI returns 406 Errors

### DISCUSSION THREAD

**Discussion by bcanzanella** [supabase/supabase#30651] [Questions]
I'm using Supabase CLI and [nextjs-user-management](https://github.com/supabase/supabase/blob/master/examples/user-management/nextjs-user-management/README.md). 

After creating a user using the Next.js auth flow, I see 406 responses after loading the account screen, as well as with any attempts to update the user's data. I have not made any changes to the codebase. Anything I can look into here? (I'm new to Supabase)

If I change my .env.local NEXT_PUBLIC_SUPABASE_URL and NEXT_PUBLIC_SUPABASE_ANON_KEY values to point to values on supabase.com (rather than local values), things work, so it seems related to a local configuration?

the response looks like:

```
{
    "code": "PGRST106",
    "details": null,
    "hint": null,
    "message": "The schema must be one of the following: "
}
```

<img width="1095" alt="Screenshot 2024-11-25 at 10 07 16 AM" src="https://github.com/user-attachments/assets/f2501866-89ce-4801-b210-8e6371d0baf5">

**Comment by silentworks** [comment]
Normally the error should tell you which schema it is making reference to. Maybe try deleting all docker container, volumes and images related to this project and then try again from scratch.

**Comment by bcanzanella** [comment]
hmm, could it be it is because the schema is empty in the example project's toml file? the comment mentions `public and storage are always included.`  https://github.com/supabase/supabase/blob/master/examples/user-management/nextjs-user-management/supabase/config.toml#L10

**Reply by silentworks** [reply]
That is it. I think this was created before those schemas had to be stated explicitly when the CLI would have exposed them by default implicitly. You can create a PR to update the example or I'll try and get one done later today or in the week.

**Reply by bcanzanella** [reply]
I opened a PR here: https://github.com/supabase/supabase/pull/30718 

thanks!
